static/css/index.css

```
@@ -2109,29 +2109,31 @@ body.react-chat-window-dragging {
         min-height: 0;
     }
 
+    /* 窄屏与桌面统一用 grid 布局：auto(顶栏) 1fr(消息) auto(输入) +
+       overflow:hidden 保证输入区不会被消息挤出。shell 尺寸仍由
+       syncMobileContentLayout() 写成显式像素（根据内容 50vh 上限）。*/
```

Stylelint (17.6.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)